### PR TITLE
Upload DSSE document to Ent and print its hash

### LINKED ISSUE #3339: Upload the DSSE to ent in the Build Provenance workflow
I wonder if we should consider also automatically uploading the provenance to ent at some point, so that we don't even have to interact with rekor at the time when we are trying to download the binary and its provenance. Anyways, let's discuss separately

_Originally posted by @tiziano88 in https://github.com/project-oak/oak/pull/3314#discussion_r995624992_

### PR DESCRIPTION
**Pull request by rbehjati**
Towards #3339

### CODE REVIEW

File: .github/workflows/provenance.yaml [hunk 1]
```diff
@@ -158,3 +158,38 @@ jobs:
...
+          ent
+          cat <<EOF > ~/.config/ent.toml
+          [[remotes]]
+          name = 'ent-store'
+          url = '${ENT_URL}'
+          write = true
+          api_key = '${{ secrets.ENT_API_KEY }}'
+          EOF
+
+      # TODO(#3339): publish as a comment
```
Comment (mariaschett) [inline .github/workflows/provenance.yaml:189]: Mhm... I am not sure what you mean by publish as a comment. I might miss some context here.

**Review by mariaschett — APPROVED** [review]

File: .github/workflows/provenance.yaml (hunk 1 above)
Comment (rbehjati) [reply .github/workflows/provenance.yaml:189]: I mean the same way we publish the reproducibility index as a comment on the PR. That would make it easier in the future to fetch the DSSE document from Ent. 

I'll fix this immediately after merge :)

**Review by rbehjati — COMMENTED** [review]
Thanks for the quick review.

File: .github/workflows/provenance.yaml (hunk 1 above)
Comment (rbehjati) [reply .github/workflows/provenance.yaml:189]: I want to publish the sha256sum as a comment. I think retrieving it from the comment would be easier than retrieving it from the logs.

**Pull request merged**